src/btree/bt_read.c

```diff
@@ -152,6 +152,78 @@ __instantiate_birthmarks(WT_SESSION_IMPL *session, WT_REF *ref)
     return (ret);
 }
 
+/*
+ * __wt_modify_vector_init --
+ *     Initialise a modify vector.
+ */
+void
+__wt_modify_vector_init(WT_MODIFY_VECTOR *modifies, WT_SESSION_IMPL *session)
```

Comment:
It would be nice if these weren't in `bt_read.c`;  maybe `support/modify.c` or `include/misc.i`?

Ignore if you like it where it is.

Reply:
885a5d8